docs/components/typography/index.html

```diff
@@ -34,6 +35,7 @@ <h3>Heading Three</h3>
       <h4>Heading Four</h4>
       <h5>Heading Five</h5>
     </div>
+  </section>
```

Comment:
This creates invalid markup. You've closed sections without opening new ones. To remediate:

1. Revert the hierarchy changes
2. Add `[data-visreg]` attributes to `div.demo` elements instead

Reply:
I see so you can't nest `section` tags like this? Interesting.

I didn't want to capture the output below the visual examples, such as description text. Seemed like that wasn't as useful as capturing just the content of the rendered demo part, not the entire demo itself.

Reply:
If you target the `<div>` elements with the `.demo` class, it should avoid the code snippets.